src/enums.h

```diff
@@ -355,4 +355,10 @@ inline bool operator<(const tripoint &a, const tripoint &b)
 static const tripoint tripoint_min { INT_MIN, INT_MIN, INT_MIN };
 static const tripoint tripoint_zero { 0, 0, 0 };
 
+enum class VisitResponse {
+    Abort, // Stop processing after this node
+    Next,  // Descend vertically to any child nodes and then horizontally to next sibling
+    Skip   // Skip any child nodes and move directly to the next sibling
```

Comment:
We should get some coding standard regarding enums.
I use ALL_CAPS, but some enums (for example, body parts) are lowercase.

Does this enum need to be declared this globally? inventory.h and character.h both include item.h - it could be defined there.
If it was ever needed, moving it from item.h to enums.h would be trivial, as all relevant files include enums.h anyway.